pkg/cmd/cli/cli.go

```diff
@@ -208,9 +202,20 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	}
 	cmds.AddCommand(cmd.NewCmdOptions(out))
 
+	setOutput(cmds, out)
+
 	return cmds
 }
 
+func setOutput(cmd *cobra.Command, out io.Writer) {
```

Comment:
This is a pretty drastic change - can you explain why this is necessary?  Why would we not just require that all of the commands set output properly?